### PR TITLE
Styling redo

### PR DESCRIPTION
**Pull request by gmaniago**
credits page styled. screenshot on slack.

### CODE REVIEW

File: styles/_credits.scss [hunk 1]
```diff
@@ -1,8 +1,11 @@
 .credits-container {
-    color: #000;
-    margin: 0;
-    padding: 0;
-
+    color: $paragraphColor;
+    widht: 90%;
```
Comment (Jmingus) [inline styles/_credits.scss]: typo on width prop

**Comment by Jmingus** [comment]
Otherwise looks good :+1:

**Comment by Joseph-A-Muela** [comment]
:+1:

File: styles/_credits.scss (hunk 1 above)
Comment (gmaniago) [inline styles/_credits.scss]: thanks. I am styling the login right now. I'll create a pull request for both.

**Comment by bobmccarthy** [comment]
:+1:

**Pull request merged**